dashboard/src/features/authn/actions.js

```diff
@@ -9,22 +9,28 @@ export const logIn = (token) => (dispatch) => {
 }
 
 export const getAuthenticationStatus = () => {
-  return (dispatch) => {
+  let connectedToCore = false
+  const pingAuth = (dispatch) => {
     return unauthedClient().config.info()
       .then(() => {
+        connectedToCore = true
         dispatch({type: 'CONNECTED_TO_CORE'})
       }, (err) => {
         if (err.status < 500) {
+          connectedToCore = true
           dispatch({type: 'CONNECTED_TO_CORE'})
         } else {
           dispatch({type: 'CORE_DISCONNECT'})
+          setTimeout(() => pingAuth(dispatch), 1000)
         }
 
         if (err.status == 401) {
           dispatch({type: 'AUTHENTICATION_REQUIRED'})
         }
       })
-      .then(() => dispatch(fetchCoreInfo()))
-      .then(() => dispatch({type: 'AUTHENTICATION_READY'}))
+      .then(() => connectedToCore && dispatch(fetchCoreInfo()))
+      .then(() => connectedToCore && dispatch({type: 'AUTHENTICATION_READY'}))
```

Comment:
This seems like a race condition between `fetchCoreInfo` and any other action that can update the flash message. This may or may not be an issue with other flash messages as well, but I just wanted to double-check.